packages/features/ee/organizations/lib/onboarding/__tests__/BaseOnboardingService.test.ts

```diff
@@ -0,0 +1,249 @@
+import { describe, expect, it } from "vitest";
+
+import { UserPermissionRole } from "@calcom/prisma/enums";
+
+import { BaseOnboardingService } from "../BaseOnboardingService";
+import type { CreateOnboardingIntentInput } from "../types";
+
+class TestableBaseOnboardingService extends BaseOnboardingService {
+  async createOnboardingIntent(input: CreateOnboardingIntentInput): Promise<any> {
+    throw new Error("Not implemented");
+  }
+
```

Comment:
Create a testable base service to ensure `testFilterTeamsAndInvites` works as we expect by calling the base services `filterTeamsAndInvites`

No need for the `createOnboardingIntent` fn due to the fact these are tested in their respective implementation classes